packages/components/src/CardFlip.tsx

```diff
@@ -42,17 +44,21 @@ const CardFlip = ({
     }
   }, []);
 
-  useEffect(() => {
-    if (!front.current) throw Error('ref undefined');
-    front.current.addEventListener('transitionstart', transitionStart);
-  }, [transitionStart]);
+  useEffect(
+    function setIsFlippingClassOnTransitionStart() {
+      if (!front.current) throw Error('ref undefined');
+      front.current.addEventListener('transitionstart', transitionStart);
```

Comment:
Just noticed that we're not cleaning up this listener - should return a cleanup function here as well that removes the `transitionstart` listener.